File: control-assurance-sp/src/types/IAPActionDirectorate.ts
import { IEntity } from "./Entity";

export interface IIAPActionDirectorate extends IEntity {
    IAPUpdateId?: number;
    DirectorateId?: number;
}

export class IAPActionDirectorate implements IIAPActionDirectorate {
    public ID = 0;
    public Title = null;
    public IAPActionId = null;
    public DirectorateId = null;
}

File: ControlAssuranceAPI/Models/IAPActionDirectorate.cs
using System;
using System.Collections.Generic;

namespace CAT.Models;

public partial class IAPActionDirectorate
{
    public int ID { get; set; }

    public string? Title { get; set; }

    public int? IAPActionId { get; set; }

    public int? DirectorateId { get; set; }

    public virtual Directorate? Directorate { get; set; }

    public virtual IAPAction? IAPAction { get; set; }
}
